coap/Kconfig

```diff
@@ -19,7 +19,7 @@ menu "CoAP Configuration"
 
     endchoice #COAP_MBEDTLS_ENCRYPTION_MODE
 
-    config COAP_MBEDTLS_DEBUG
+    config COAP_DEBUGGING
```

Comment:
Suggest dropping `sdkconfig.rename` file here to maintain backward compatibility. Please see template here: https://github.com/espressif/esp-idf/blob/9a1cc59338aa151bc6333f85c8169d36a478fe2a/sdkconfig.rename#L1-L2

Reply:
I assume you mean adding in a sdkconfig.reame file in the coap directory?
I changed the name for clarity.

Reply:
> I assume you mean adding in a sdkconfig.reame file in the coap directory?

Yes, please.

Reply:
Done